docs/source/guide/get_started/installation.rst

```diff
@@ -79,71 +79,23 @@ Install OpenVINO™ Training Extensions for users (XPU devices)
 *************************************************************
 
 1. Install OpenVINO™ Training Extensions
-package:
-
-* A local source in development mode
+from source to use XPU functionality.
 
 .. tab-set::
 
-    .. tab-item:: PyPI
-
-        .. code-block:: shell
-
-            pip install otx
-
-    .. tab-item:: Source
+    .. tab-item:: Minimum requirements
 
         .. code-block:: shell
 
             # Clone the training_extensions repository with the following command:
             git clone https://github.com/openvinotoolkit/training_extensions.git
             cd training_extensions
 
-            # Set up a virtual environment.
-            python -m venv .otx
-            source .otx/bin/activate
-
-            pip install -e .
-
-2. Install Intel Extensions for Pytorch & Requirements
-for training according to your system environment.
-
-.. tab-set::
-
-    .. tab-item:: Minimum requirements
-
-        .. code-block:: shell
-
-            pip install '.[xpu]' --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/
+            pip install -e '.[xpu]' --extra-index-url https://download.pytorch.org/whl/test/xpu
 
-        .. note::
+[Optional] Refer to the `PyTorch documentation guide <https://pytorch.org/docs/stable/notes/get_start_xpu.html>`_
 
-            Models from mmlab are not available for this environment. If you want to use mmlab models, you must install them with Full Requirements.
-            Also, some tasks may not be supported by minimum requirements.
-
-    .. tab-item:: Full Requirements
-
-        .. code-block:: shell
-            python -m pip install torch==2.1.0.post2 torchvision==0.16.0.post2 torchaudio==2.1.0.post2 intel-extension-for-pytorch==2.1.30+xpu oneccl_bind_pt==2.1.300+xpu --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/
-            git clone https://github.com/open-mmlab/mmcv
-            cd mmcv
-            git checkout v2.1.0
-            MMCV_WITH_OPS=1 pip install -e .
-            cd ..
-            otx install -v --do-not-install-torch
-
-[Optional] Refer to the `Intel® Extension for PyTorch documentation guide <https://intel.github.io/intel-extension-for-pytorch/index.html#installation?platform=gpu&version=v2.1.10%2bxpu&os=linux%2fwsl2&package=pip>`_
```

Comment:
I'd add https://www.intel.com/content/www/us/en/developer/articles/tool/pytorch-prerequisites-for-intel-gpu/2-5.html here, because the drivers and related runtime binaries should be installed as well

Reply:
It is already outlined in the PyTorch documentation as the first step